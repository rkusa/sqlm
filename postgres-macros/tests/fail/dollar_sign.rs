fn main() {
    sqlm_postgres_macros::sql!("SELECT {id} = $1", id = 1);
}
