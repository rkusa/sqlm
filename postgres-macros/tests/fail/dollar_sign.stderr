error: use {} instead of $x for positional parameters
 --> tests/fail/dollar_sign.rs:2:32
  |
2 |     sqlm_postgres_macros::sql!("SELECT {id} = $1", id = 1);
  |                                ^^^^^^^^^^^^^^^^^^
